tests/integration/adapter/find-all-test.js

```diff
@@ -19,6 +19,7 @@ module("integration/adapter/find_all - Finding All Records of a Type", {
       firstName: attr('string'),
       lastName: attr('string')
     });
+    Person.reopenClass({ toString: () => 'Person' });
```

Comment:
I am curious why this is needed. Which tests are failing without this changes? I would guess only tests wich check for assertions, right? Maybe we should only specify the `toString` in those tests...